config/server-vars.yml

```diff
@@ -108,3 +108,15 @@ EDXAPP_EMAIL_HOST: "%%EDXAPP_EMAIL_HOST%%"
 
 # Default log levels: DEBUG, INFO, WARNING, ERROR, CRITICAL
 #EDXAPP_LOG_LEVEL: 'INFO'
+
+EDXAPP_LMS_NGINX_PORT: 18000

```

Comment:
Yes. As discussed, this edxapp role default gets overridden in fullstack and sandbox playbooks